docs/FAQ.md

```diff
@@ -0,0 +1,74 @@
+# Frequently Asked Questions (FAQs)
+
+## Users
+**Q : What is fAIr?**
+
+A : fAIr is an open-source toolkit developed by the Humanitarian OpenStreetMap Team (HOT) that enables the integration of artificial intelligence (AI) into humanitarian mapping workflows. It provides AI models and tools to automate mapping tasks, improving efficiency and accuracy.
+
+<hr>
+
+**Q : How can I use fAIr in my mapping projects?**
+
+A : To use fAIr, you can follow the installation instructions provided in the documentation. Once installed, you can utilize the pre-trained AI models for tasks such as road network extraction, building footprint detection, or land use classification. The documentation offers examples and guidelines to help you get started.
+
+<hr>
+
+**Q : What types of data can I use with fAIr?**
+
+A : fAIr primarily works with satellite imagery and aerial photographs. You can use various types of imagery sources, including public satellite imagery datasets, to perform mapping tasks using the AI models provided. The documentation provides information on the compatible data formats and recommended data sources.
+
+<hr>
+
+**Q : Can I customize the AI models in fAIr for my specific needs?**
+
+A : Yes, fAIr is designed to be customizable. The toolkit provides instructions and guidelines on how to train and fine-tune models with your own datasets. This allows you to adapt the models to suit your specific mapping requirements and improve their performance on your target areas or features of interest.
+
+<hr>
+
+**Q : Are there any limitations to using fAIr?**
+
+A : While fAIr offers powerful AI models, it's important to note that the accuracy of the results may vary depending on the quality and resolution of the input imagery and the complexity of the mapping task. Some tasks may require additional manual editing or validation. It's recommended to review the documentation and understand the limitations of each model before use.
+
+<hr>
+
+**Q : How can I contribute feedback or report issues with fAIr?**
+
+A : You can contribute to the improvement of fAIr by providing feedback, reporting issues, or suggesting new features through the project's GitHub repository. By creating an issue, you can share your observations or difficulties faced while using fAIr, allowing the developers to address them and enhance the toolkit.
+
+<hr>
+
+**Q : Can I integrate fAIr with other mapping tools or platforms?**
+
+A : Yes, fAIr is designed to be modular and can be integrated into existing mapping workflows and platforms. It provides interoperability with tools like OpenStreetMap, QGIS, and ArcGIS. You can leverage the AI models from fAIr within these tools or use the outputs of fAIr in combination with other mapping datasets and tools.
```

Comment:
It provides modular pip packages/ API endpoints that can be used in other scripts and can be integrated as plugins on different platforms such as Qgis, Josm etc

Reply:
Done